### PR TITLE
Fix form layout in Car price calculator

### PR DESCRIPTION
**Pull request by robinandeer**
## Describe your changes

Fix layout issue in price form for SE car.

Fix (again...) the animaiton hook to highlight inputs as you type.

## Justify why they are needed

The hook was ignoring when you called "highlight" wihout an event.

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1101** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1101" target="_blank">Graphite</a> 
    * **PR #1102** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1102" target="_blank">Graphite</a> 
      * **PR #1103** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1103" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1103?utm_source=stack-comment).

File: apps/store/src/services/PriceCalculator/data/SE_CAR.ts [hunk 1]
```diff
@@ -32,8 +32,8 @@ export const SE_CAR: Template = {
       title: { key: 'SECTION_TITLE_YOUR_ADDRESS' },
       submitLabel: { key: 'SUBMIT_LABEL_FINISH' },
       items: [
-        { field: streetAddressField, layout: { columnSpan: 4 } },
-        { field: postalCodeField, layout: { columnSpan: 2 } },
+        { field: streetAddressField, layout: LAYOUT.FULL_WIDTH },
+        { field: postalCodeField, layout: LAYOUT.FULL_WIDTH },
```
Comment (alebedev) [inline apps/store/src/services/PriceCalculator/data/SE_CAR.ts:36]: Maybe half width?  Full line for postal code is probably too much

**Review by alebedev — APPROVED** [review]
Looks good

File: apps/store/src/services/PriceCalculator/data/SE_CAR.ts (hunk 1 above)
Comment (robinandeer) [reply apps/store/src/services/PriceCalculator/data/SE_CAR.ts:36]: The "address" label/input doesn't fit :P

But let's discuss with designers later :)

**Comment by robinandeer** [comment]
**Graphite Merge Job**

Current status: ⏳ Queued to merge

This pull request is currently queued to be merged as part of a stack.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1103?utm_source=stack-comment).


Job Reference: SIFJxfkJlWZjfGeII3OL

**Pull request merged**